src/schema/v2/__tests__/marketingCollections.test.ts

```diff
@@ -172,4 +172,48 @@ describe("MarketingCollections", () => {
       curatedMarketingCollections: payload,
     })
   })
+
+  it("uses the custom sort for marketing collections", async () => {
+    const query = gql`
+      {
+        marketingCollections(sort: CUSTOM, size: 2, category: "Gallery") {
+          slug
+        }
+      }
+    `
+
+    const payload = [
+      {
+        slug: "percys-z-collection-1",
+      },
+      {
+        slug: "fiby-z-collection-2",
```

Comment:
**question:** Who is Fiby??

<img width=100 src=https://github.com/user-attachments/assets/268cb887-f04d-4722-a2d9-37449c742416 />

Reply:
![telegram-cloud-photo-size-4-5841689928617019050-y](https://github.com/user-attachments/assets/ba4d2c34-e9a3-4c31-a9ff-1edc9fa9f70a)